### PR TITLE
Update README to include more details of usage and dependency info

### PR DESCRIPTION
**Pull request by tai271828**
## Types of changes

- **Documentation Update**

## Description

Update README to include more details of usage and dependency info.

The script has been tested with Ubuntu 20.04.2 rc image on a Bionic host.

## Checklist:
- [ ] Add test cases to all the changes you introduce
- [ ] Run `poetry run pytest` locally to ensure all linter checks pass
- [x] Update the documentation if necessary

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -26,6 +37,8 @@ Today `ubuntu-server-netboot` needs to run on Ubuntu or another Debian derivativ
  - pxelinux (x86-only)
  - syslinux-common (x86-only)
 
+This script is tested with Bionic and above.
```
Comment (dannf) [inline README.md]: *nit* - can you say Ubuntu 18.04 ('bionic') and above? I'm not sure if we can expect the average community member to know Ubuntu codenames.

File: README.md [hunk 2]
```diff
@@ -17,6 +17,17 @@ If you have a local copy of the ISO, you can point to it with the `--iso` parame
 
 You can also add additional kernel command line arguments (e.g. `"console=ttyS0"`) to the generated configuration files using the `--extra-args` parameter.
 
+## Usage
+
+Copy the files generated under the interim folder `/tmp/tmpxxx/ubuntu-installer/`
+to your tftp root folder for netboot, for example `/var/lib/tftpboot`:
+
```
Comment (dannf) [inline README.md:23]: Yes, thanks. I was hoping that you could actually `mv /tmp/tmpxxx/ubuntu-installer /var/lib/tftpboot`, and then use `ubuntu-installer/grubnet<arch>.efi` as the filename. i.e., use the `ubuntu-installer` namespace. But when I tried this, `GRUB` wasn't able to find the grub.cfg file :( I thought this used to work with `d-i`, but maybe the `grubnet<arch>.efi` it provides has a different built-in config?)

File: README.md [hunk 3]
```diff
@@ -17,6 +17,17 @@ If you have a local copy of the ISO, you can point to it with the `--iso` parame
...
+```
+$ sudo cp /tmp/tmpxxx/ubuntu-installer/* /var/lib/tftpboot/
+```
+
+Then your netboot server is ready to go.
+
```
Comment (dannf) [inline README.md:35]: We should probably note that you still need to setup a DHCP server - though we can probably just point to an external document on that (IIRC, Ubuntu discourse describes this?)

**Review by dannf — CHANGES_REQUESTED** [review]
Thanks @tai271828 ! Some feedback inline

File: README.md (hunk 2 above)
Comment (tai271828) [reply README.md:23]: Using `ubuntu-installer` namespace is making more sense so I tried this as well. And right, `grubnet<arch>.efi` always looks for the root folder of tftp and ubuntu-installer namespace does not work : (

File: README.md (hunk 3 above)
Comment (tai271828) [reply README.md:35]: Thanks for the reminder. Yes, [the discourse page](https://discourse.ubuntu.com/t/netbooting-the-live-server-installer-via-uefi-pxe-on-arm-aarch64-arm64-and-x86-64-amd64/19240) describes this. This tool (and its README) will be updated in the discourse page, and it is a good idea to refer to the discourse page so they could mutually refer to each other.

**Review by dannf — APPROVED** [review]
I've a few minor changes, but I'll propose those as my own MP if you don't mind reviewing..

**Pull request merged**